server/config/config.js

```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
```

Comment:
Why not use es6 imports?

Reply:
I'm having issues using es6 imports and exports from sequelize